lagom-client-restaurant/menu-item-impl/src/main/resources/application.prod.conf

```diff
@@ -2,63 +2,18 @@ include "application.conf"
 
 play.http.secret.key = "unused secret"
 
-http {
-  address = ${?HTTP_BIND_ADDRESS}
-  port = 9000
-}
-
-play.filters.hosts {
-  # Requests that are not from one of these hosts will be rejected.
-  allowed = [${?ALLOWED_HOST}]
-}
```

Comment:
This didn't seem to be necessary. I think Lagom doesn't install the allowed hosts filter by default as Play does.